test/azure/generated/AzureCompositeModelClient/azureCompositeModel.ts

```diff
@@ -47,24 +45,27 @@ class AzureCompositeModel extends AzureCompositeModelContext {
     this.readonlyproperty = new operations.Readonlyproperty(this);
     this.flattencomplex = new operations.Flattencomplex(this);
   }
-  // methods on the client.
 
   /**
    * The Products endpoint returns information about the Uber products offered at a given location.
    * The response includes the display name and other details about each product, and lists the
    * products in the proper display order.
-   *
    * @summary Product Types
-   *
    * @param resourceGroupName Resource Group ID.
-   *
-   * @param [options] Optional Parameters.
-   *
-   * @returns A promise is returned
+   * @param [options] The optional parameters
+   * @returns Promise<Models.ListResponse>
+   */
+  list(resourceGroupName: string, options?: msRest.RequestOptionsBase): Promise<Models.ListResponse>;
+  /**
```

Comment:
What's the value added for the doc comments with no description on the overloads, e.g. in the editor or in docs.microsoft.com?

Reply:
I don't think I'd want to copy the doc around because it can bloat the .d.ts file, just wondering what documenting the parameters specifically does.

Reply:
The value is that we don't duplicate the description in every overload. The idea is that in your editor the description is shown while you're choosing a method, but you don't need to see the description again after you've chosen the method. At that point you only need to see the parameters and the return type.